### PR TITLE
Restructure the Ubuntu Server USB disk installation workflow

### LINKED ISSUE #337: Tell people to start Ubuntu download earlier
## Describe the change
Downloading Ubuntu over Tails/Tor took me an hour. I could've started it like midway through the firewall setup (basically as soon as the admin workstation has network connectivity) and wouldn't have had to spend that long waiting.

Alternatively, if we have another computer we trust, like the one that is creating the initial Tails USB, we could download Ubuntu over the clearnet, since there's really no privacy gains from downloading Ubuntu over Tor.

### PR DESCRIPTION
**Pull request by nathandyer**
## Status

Ready for review

## Description of Changes

This branch completely restructures the Ubuntu Server boot disk portion of the SecureDrop docs. Rather than including that in the server setup page, and having users download from the Admin Workstation over the TOR network (which is quite slow), the Tails Setup page has been made into a more comprehensive `Create USB Boot Disks` page that covers both Tails and Ubuntu Server. Doing it this way has two advantages:

1. It speeds up the Ubuntu Server download itself (there's not a significant security advantage to downloading the Ubuntu Server ISO over TOR)
2. It makes it easier to create the Ubuntu Server USB disk because the user is already using a tool like `GNOME Disks` to flash an image onto the USB drive for the Tails steps above

An additional change is that rather than documenting creating the Ubuntu Server disk with `GNOME Disks` specifically, it now links out to Canonical's documentation for Ubuntu, and has guides for Windows, Mac, and Linux. This reduces our documentation maintenance burden for those steps.

Another, smaller portion of these changes relate to updating references to these sections in other documents. These are generally one-line changes.

* Resolves #337

* Related Issues

## Testing

* Checkout the branch and build the docs using the instructions in the readme. The most relevant pages to check are `Create USB Boot Disks` and `Set Up The Servers`

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [x] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/create_usb_boot_disks.rst [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+Overview
+-------------
+For the initial SecureDrop setup, you will need three USB disks.
+
+Two of them will will be used for the Tails operating system,
+for creating the *Admin Workstation* and *Secure Viewing Station*.
+
+The other USB disk will have the Ubuntu Server installer,
+which is needed to install the underlying server OS for your
+*application* and *monitor* servers.
```
Comment (eloquence) [inline docs/create_usb_boot_disks.rst:13]: Nit: capitalize and italicize the entire term (i.e. *Application* and *Monitor Servers*).

File: docs/create_usb_boot_disks.rst [hunk 2]
```diff
@@ -0,0 +1,214 @@
+Create USB Boot Disks
```
Comment (eloquence) [inline docs/create_usb_boot_disks.rst:1]: Nit: I would prefer to standardize on using "USB stick" or "USB drive" (we don't use "USB disk" anywhere else in the docs; Google indicates it's the least common of the three).

File: docs/create_usb_boot_disks.rst [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+The other USB disk will have the Ubuntu Server installer,
+which is needed to install the underlying server OS for your
+*application* and *monitor* servers.
+
+Although the Ubuntu Server disk will be used later in the setup process,
+we recommend downloading and creating the disk at the same time you create
+the Tails USB disks to avoid having to repeat steps, and to avoid slower
+download speeds later that would be introduced by trying to download Ubuntu
+over the TOR network from the *Admin Workstation.*
```
Comment (eloquence) [inline docs/create_usb_boot_disks.rst:19]: I think you can save this entire paragraph; you're already explaining what the purpose of the Ubuntu disk is in the preceding paragraph, and we can reasonably assume that most folks will just follow the instructions in the order in which we present them.

File: docs/create_usb_boot_disks.rst [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+done from computers using Tails, so the first thing you need to do is
+set up several USB drives with the Tails operating system. To get
+started, you'll need two Tails drives: one for the *Admin Workstation*
+and one for the *Secure Viewing Station*. :doc:`Later <onboarding_journalists>`,
+you'll set up a bunch more Tails drives for your journalists and
+backups, but for now you just need two.
+
+As soon as you create a new Tails drive, *label it immediately*. USB
+drives all look alike and you're going to be juggling a whole bunch of
+them throughout this installation. Label immediately. Always.
```
Comment (eloquence) [inline docs/create_usb_boot_disks.rst:41]: How about generalizing this and moving it into the first section?

File: docs/create_usb_boot_disks.rst [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+
+.. _download_ubuntu:
+
+Download the Ubuntu Installation Media
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The installation media and the files required to verify it are available on the
+`Ubuntu Releases page`_. You will need to download the following files:
+
+* `ubuntu-20.04.4-live-server-amd64.iso`_
```
Comment (eloquence) [inline docs/create_usb_boot_disks.rst:134]: Heads up: It looks like 20.04.5 is around the corner (September 1 according to https://wiki.ubuntu.com/Releases). We'll want to point to that version consistently once it's live.

**Review by eloquence — CHANGES_REQUESTED** [review]
Thanks for this reorg, makes sense. :) A few small comments in review.

File: docs/create_usb_boot_disks.rst (hunk 5 above)
Comment (nathandyer) [reply docs/create_usb_boot_disks.rst]: Just a note that although it's not showing up there in the Wiki yet, Ubuntu 20.04.5 was released earlier today. The links are now active, so I have updated everything.

**Review by eloquence — APPROVED** [review]
Looks great! :)

**Pull request merged**